provider/build.gradle

```diff
@@ -71,4 +76,4 @@ dependencies {
     androidTestImplementation 'com.android.support.test.espresso:espresso-core:3.0.2'
 }
 
-apply from: 'gradle-mvn-push.gradle'
\ No newline at end of file
+// apply from: 'gradle-mvn-push.gradle'
```

Comment:
Can we remove this all together or write a code comment as to why it's commented out

Reply:
No, I just commented it out because this contains all the gradle tasks for publishin the library builds to repo servers and that flow doesn't work at the moment. But it will be brought back.

Reply:
Ideally would be good to add that as a code comment but if it's in near future (upcoming PR?) then no worries :)